packages/sui-studio/bin/helpers/cli.js

```diff
@@ -0,0 +1,40 @@
+
+/**
+ * Calls sui-mono comman with sui-studio default config
+ * @param  {String} command Name of the command
+ * @return {ChildProcess}
+ */
+function callSuiMonoCommand (command) {
+  const BIN_PATH = require.resolve('@schibstedspain/sui-mono/bin/sui-mono')
+  callCommand(BIN_PATH, [ command,
+    ...getSuiMonoOptions()
+  ])
+}
+
+/**
+ * Parses sui-mono config in package.json and returns corresponding CLI options
+ * @return {Array<String>} CLI options
+ */
+function getSuiMonoOptions () {
+  const path = require('path')
+  const pkg = require(path.join(__dirname, '..', '..', 'package.json'))
+  const conf = pkg && pkg.config && pkg.config['sui-mono']
```

Comment:
Well, I've reverted these commits...
The options thing was getting more complicated when sui-cz was involved...
So, we'll do that sui-studio-create add "sui-mono" config in package.json. Otherwise, code would be messy.